README.md

```diff
@@ -1 +1,131 @@
-# book-store
\ No newline at end of file
+# todo-react
+<a name="readme-top"></a>
+
+
+
+<div align="center">
+
+  <h3><b>todo-react</b></h3>
+
+</div>
+
+
+# 📗 Table of Contents
+
+- [todo-react](#todo-react)
```

Comment:
- [ ] Let's remove all the references to the 'todo' app in the README as this is the bookstore app. Cheers 👍